lib/datadog/tracing/span_operation.rb

```diff
@@ -383,15 +378,24 @@ def initialize
 
         # Triggered when the span raises an error during measurement.
         class OnError < Tracing::Event
-          def initialize
+          def initialize(default)
```

Comment:
I like this "default" behavior in the constructor. Maybe something we should consider adding to other events where appropriate later.